serial.md

```diff
@@ -63,13 +63,14 @@ only be specified if the platform requires them to be set.
 ```
 0  START_SYSEX      (0xF0)
 1  SERIAL_DATA      (0x60)  // command byte
-2  SERIAL_CONFIG    (0x10)  // OR with port (0x11 = SERIAL_CONFIG | HW_SERIAL1)
-3  baud             (bits 0 - 6)
-4  baud             (bits 7 - 13)
-5  baud             (bits 14 - 20) // need to send 3 bytes for baud even if value is < 14 bits
-6  rxPin            (0-127) [optional] // only set if platform requires RX pin number
-7  txPin            (0-127) [optional] // only set if platform requires TX pin number
-6|8 END_SYSEX      (0xF7)
+2  SERIAL_CONFIG    (0x10)
```

Comment:
This preserves the original constants. It's kinda weird when adding new constants since they'll probably have to start with `0x00` or `0x01`.